src/e2e/__tests__/caret.ts

```diff
@@ -39,7 +36,14 @@ describe('caret testing', () => {
 
     await page.keyboard.press('Enter')
     await paste(page, [''], importText)
-    await setCursor(page, ['puppeteer', 'web scrapping'], { offset: 3 })
+
+    await clickBullet(page, 'puppeteer')
+    await clickBullet(page, 'web scrapping')
+
+    await waitForEditable(page, 'web scrapping')
+    const editableNodeHandle = await getEditable(page, 'web scrapping')
+    await clickWithOffset(page, editableNodeHandle, { horizontalClickLine: 'left', offset: 3 })
```

Comment:
`offset` overrides `horizontalClickLine`, so in this case you can omit `horizontalClickLine`.